services/movie_session.py

```diff
@@ -42,3 +42,13 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
The same issue with the `delete_movie_session_by_id` function - it does not handle the case when the `MovieSession` with the provided id does not exist. It's a bug.